### PR TITLE
feat: convert heic images to jpg

### PR DESCRIPTION
**Pull request by sudoDeznit**
Converts heic images to jpg using react-native-heic-converter package.
conversion happens when user selects image from gallery.
an extra field path is added with uri to .jpg file

**Note**
Needs npm & pod install
Needs proper testing on real device.. specially uninstall old app and then try to see if any permission issue occurs

### CODE REVIEW

File: screens/home/HomeScreen.js [hunk 1]
```diff
@@ -543,7 +549,7 @@ class HomeScreen extends PureComponent {
                     galleryToUpload.append('date', date);
                     galleryToUpload.append('presence', img.picked_up);
                     galleryToUpload.append('model', model);
-
+                    console.log(JSON.stringify(galleryToUpload, null, 2));
```
Comment (xlcrr) [inline screens/home/HomeScreen.js:552]: Log can be removed

**Comment by xlcrr** [comment]
Nice work. Maybe this should happen during upload? Testing now

File: screens/home/HomeScreen.js (hunk 1 above)
Comment (sudoDeznit) [reply screens/home/HomeScreen.js:552]: kept it so that we know from sentry logs if something goes wrong...

**Comment by sudoDeznit** [comment]
tried at upload.. felt slow.. and when user upload multiple images it will add up..

**Pull request closed without merging**